src/lavinmq/journal_error_writer.cr

```diff
@@ -0,0 +1,15 @@
+module LavinMQ
+  class JournalErrorWriter < IO::FileDescriptor
+    def initialize
+      super(STDERR.fd, blocking: STDERR.blocking)
+    end
+
+    def write(slice : Bytes) : Nil
+      self.write_byte(60_u8) # '<'
+      self.write_byte(51_u8) # '3'
+      self.write_byte(62_u8) # '>'
+      self.write_byte(32_u8) # ' ' (space)
+      super slice
```

Comment:
If `STDERR.sync? == true` this will be four syscalls (five with "super"). It should be fine to do `self.write_string "<3> "`.